upgrade-guide.md

```diff
@@ -64,6 +66,34 @@ or
 auto new_edged_curve = geode::load_edged_curve< 3 >( geode::OpenGeodeEdgedCurve3D::type_name_static(), filename );
 ```
 
+- **Mesh & Builder factories**: Mesh and MeshBuilder factories has been merge into one.
```

Comment:
```suggestion
- **Mesh & Builder factories**: Mesh and MeshBuilder factories has been merged into one.
```

Reply:
There is also breaks in inheritance between e.g. PolygonalSurface and TriangulatedSurface, that needs client code updates